schemas/dex/SCHEMA.md

```diff
@@ -0,0 +1,186 @@
+# DEX Schema
+
+This is OpenBlock Labs standard DEX schema.
+
+## Schema
+
+### Pools
+| Property              | Description                                                          | Type   |
+|-----------------------|----------------------------------------------------------------------|--------|
+| chain_id              | The standard id of the chain.                                        | number |
+| creation_block_number | The block this pool was created in.                                  | number |
+| creation_timestamp    | The timestamp of the block that this pool was created in.            | number |
+| pool_address          | The contract address of the lp pool.                                 | string |
+| lp_token_address      | The token address of the LP token for this pool.                     | string |
+| lp_token_symbol       | The symbol of the LP token.                                          | string |
+| token_address         | The token address of the pool token at token_index.                  | string |
+| token_symbol          | The symbol of the pool token.                                        | string |
+| token_decimals        | The decimals of the pool token.                                      | string |
+| token_index           | The index in the pool smart contract that this token appears at, default 0 (ie, one entry per token in a pool).     | number |
+| fee_rate              | (Optional, if dynamic fee) The fee rate of the pool, as a percentage (ie, 2.3% as 2.3).         | number |
```

Comment:
preferred to be will be 0.023 here instead of 2.3%

its better when building quicksight dashboard or using it to calculate fees